superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -814,6 +863,25 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
         </>
       );
     }
+
+    // Import doesn't use db state, so footer will not render in the if statement above
+    if (importingModal) {
```

Comment:
is this a dupe of line 834 - 848?

Reply:
Oh whoops, good catch! Since importing databases don't use the db state, the above if statement won't allow the footer to render when importing. I had to make a separate identical footer for an importing database on 868, but I forgot to take importingModal out of the check above. I removed it in [`this commit`](https://github.com/apache/superset/pull/19314/commits/1420929e0965e06ca83f5e3df06c6d5db0a3872f).